### PR TITLE
Add package: khal

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Had to wait until vdirsyncer was merged.

Will test ASAP.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Is the build fail because of my changes?

File: pkgs/applications/misc/khal/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    sha256 = "13zm6kak45bx298rvchm7az4z7gyjw9ddzzvn91jlcrns05baxba";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    click
+    icalendar
+    urwid
+    pyxdg
+    pytz
+    vdirsyncer
```
Comment (pSub) [inline pkgs/applications/misc/khal/default.nix]: Looks like `vdirsyncer` is not in scope.

**Comment by matthiasbeyer** [comment]
Can someone tell me what went wrong here and how to fix?

**Comment by pSub** [comment]
From what I can see from the travis log, you seem to be missing the dependency `docopt`.

```Couldn't find index page for 'docopt' (maybe misspelled?)```

**Comment by matthiasbeyer** [comment]
Updated the package to 0.4.0, which removes the "docopt" dependency.

Just a quick question besides: I tested the installation via `nix-env -iA pkgs.khal -f .` from the nixpkgs root directory, but surprisingly I cannot remove the package afterwards (neither `nix-env -e khal` nor `nix-env -e khal -f .` work) - How to do this?

**Comment by matthiasbeyer** [comment]
Hm. Now things get weird. I managed to build and install the package here on my machine, so why is there a build error?

**Comment by pSub** [comment]
Sometimes travis fails randomly. Do you have building with chroot enabled?

**Comment by matthiasbeyer** [comment]
Nope, I will try.

**Comment by matthiasbeyer** [comment]
Am I right with this:

```bash
nix-build --option build-use-chroot true -A pkgs.khal -I .
```

from the nixpkgs root directory?

It just prints the output path of the derivation:

```bash
/nix/store/<hash>-python3.4-khal-0.4.0
```

?

**Comment by matthiasbeyer** [comment]
@pSub, @geerds ping.

**Pull request merged**